framework/src/packets/icmp/v6/ndp/options/prefix_info.rs

```diff
@@ -254,6 +255,20 @@ impl fmt::Display for PrefixInformation {
     }
 }
 
+impl NdpAddOptions for PrefixInformation {
+    #[inline]
+    fn push_option(mbuf: *mut MBuf, offset: usize) -> Result<Self>
```

Comment:
guessing a unit'y test is coming? @scrollins?

Reply:
yep. working on them now.

Reply:
pushed up a couple of unit tests. i'll add more in the morning for the remainder of the ndp options